Cython/Utility/ObjectHandling.c

```diff
@@ -2314,7 +2323,7 @@ static CYTHON_INLINE PyObject* __Pyx_PyObject_FastCallDict(PyObject *func, PyObj
 
     if (kwargs == NULL) {
         #if CYTHON_VECTORCALL
-        #if Py_VERSION_HEX < 0x03090000
+        #if PY_VERSION_HEX < 0x03090000
```

Comment:
There's another mis-spelling of `Py_VERSION_HEX` in Nodes.py. I'm going to add it to this PR just because it seems like a convenient time to fix it

Reply:
No I'm not because I can't work out how to do it... I'll submit it separately

Reply:
I pushed this one to master separately. It doesn't belong here.